src/sentry/migrations/0359_duplicate_perf_issue_events_issue_platform.py

```diff
@@ -0,0 +1,222 @@
+        raise ValueError(f"no event found in eventstore({project_id}, {event_id})")
+
+    # We always fetch the stored hashes here.  The reason for this is
+    # that we want to show in the UI if the forced grouping algorithm
+    # produced hashes that would normally also appear in the event.
+    hashes = event.get_hashes()
+
+    # Transactions events are grouped using performance detection. They
+    # are not subject to grouping configs, and the only relevant
+    # grouping variant is `PerformanceProblemVariant`.
+
+    from sentry.utils.performance_issues.performance_detection import EventPerformanceProblem
+
+    problems = EventPerformanceProblem.fetch_multi([(event, h) for h in hashes.hashes])
+    # TODO: not sure how this can work
+    # a transaction can have N number of issues created, resulting in N groups created
+    # we might have to do a GroupHash lookup here to find the fingerprint since we cant map them
+    # neatly from just the event
+    return [problem.problem.fingerprint for problem in problems or ()]
+
+
+def _query_performance_issue_events() -> Sequence[Mapping[str, Any]]:
+    from snuba_sdk import Column, Condition, Function, Op, Query, Request
+
+    snuba_request = Request(
+        dataset=Dataset.Transactions.value,
+        app_id="migration",
+        query=Query(
+            match=EntityKey.Transactions.value,
+            select=[
+                Function("arrayJoin", parameters=[Column("group_ids")], alias="group_id"),
+                Column("project_id"),
+                Column("event_id"),
+                Column("trace_id"),
+                Column("span_id"),
+                Column("transaction_name"),
+                Column("transaction_hash"),
+                Column("transaction_op"),
+                Column("transaction_status"),
+                Column("start_ts"),
+                Column("start_ms"),
+                Column("finish_ts"),
+                Column("finish_ms"),
+                Column("duration"),
+                Column("platform"),
+                Column("environment"),
+                Column("release"),
+                Column("dist"),
+                Column("ip_address_v4"),
+                Column("ip_address_v6"),
+                Column("user"),
+                Column("user_id"),
+                Column("user_name"),
+                Column("user_email"),
+                Column("sdk_name"),
+                Column("sdk_version"),
+                Column("http_method"),
+                Column("http_referer"),
+                Column("tags"),
+                Column("contexts"),
+                Column("measurements"),
+                Column("span_op_breakdowns"),
+                Column("spans"),
+                Column("type"),
+                Column("message"),
+                Column("title"),
+                Column("transaction_source"),
+                Column("timestamp"),
+                Column("app_start_type"),
+                Column("profile_id"),
```

Comment:
Do we actually need all the columns? Since we're just creating this perf occurrence via `event_id`, we might be able to cut down the field we need a fair bit.